### PR TITLE
Modal state flow changes

### PR DESCRIPTION
**Pull request by rperez89**
This PR address this suggestion

We should add a third step to the modal for the user to allow the 1Hive protocol

We need to make some color adjustments to differentiate more the states of "Transaction is being minded..." and "Transaction successfully mined", here you can see the differentiation. Not only the icon changes, the color of the arrow too, they should be yellow when the transaction is being minded. The text should change the color too

The "Transaction being minded" changes to "Transaction is being minded..."

The "Transaction mined" text changes to "Transaction successfully mined"

The Waiting for signature step indicator, should change the grey to our grey, now it has the Aragon's one

**from #336**

### CODE REVIEW

File: src/components/Stepper/Step/Illustration.js [hunk 1]
```diff
@@ -9,36 +10,33 @@ import {
...
   [STEP_SUCCESS]: signRequestSuccessIllustration,
   [STEP_ERROR]: signRequestFailIllustration,
 }
 
 function Illustration({ status, index }) {
+  const theme = useTheme()
   return (
     <>
-      {/* {status === STEP_WORKING ? (
+      {status === STEP_PROMPTING ? (
```
Comment (fabriziovigevani) [inline src/components/Stepper/Step/Illustration.js:25]: Might make more sense to remove this as it was before and reuse this with the div on the `StatusVisual` component and change the `renderIllustration` logic?

File: src/components/Stepper/Step/StatusVisual.js [hunk 2]
```diff
@@ -209,11 +208,11 @@ function StepIllustration({ number, status, withoutFirstStep }) {
...
             justify-content: center;
-            background-color: ${theme.surfaceOpened};
+            background-color: ${theme.surfaceIcon};
             height: 100%;
             width: 100%;
             border-radius: 100%;
-            color: ${theme.accentContent};
+            color: ${theme.positiveContent};
 
             ${textStyle('title3')}
```
Comment (fabriziovigevani) [inline src/components/Stepper/Step/StatusVisual.js:217]:
```suggestion
            ${textStyle('title3')};
```

**Review by fabriziovigevani — COMMENTED** [review]
Awesome Ro, one small comment

Also i noticed that the animated border is a bit closer to the step circle than it should

Screen Shot 2021-03-24 at 20 12 38

File: src/components/Stepper/Step/Illustration.js (hunk 1 above)
Comment (rperez89) [reply src/components/Stepper/Step/Illustration.js:25]: i thought about that but i ended thinking that makes more sense to have it in the component in charge of rendering the illustration for each status although is not a illustration itself but act as one

**Comment by rperez89** [comment]
> Awesome Ro, one small comment
> 
> Also i noticed that the animated border is a bit closer to the step circle than it should
> 
> Screen Shot 2021-03-24 at 20 12 38

Fabri not sure , this is an screenshot of the current code 
Screen Shot 2021-03-25 at 1 30 31 AM

what do you think?

File: src/components/Stepper/Step/Illustration.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/Stepper/Step/Illustration.js:25]: Ok lets leave it as is

**Review by fabriziovigevani — APPROVED** [review]
LGTM!

**Pull request merged**